services/mailer.js

```diff
@@ -1,32 +1,32 @@
-const mailgun = require('mailgun-js');
+const mailgun = require('mailgun-js')
 const logger = require('../lib/log')(__filename)
 require('dotenv').config()
 
-const mg = mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.MAILGUN_DOMAIN});
+const mg = mailgun({ apiKey: process.env.MAILGUN_API_KEY, domain: process.env.MAILGUN_DOMAIN })
 
 const mgModule = {}
 
-mgModule.sendConfirmationEmail = (receiver, token) => {    
-    const link = `https://learndatabases.dev/emailConfirmation/${token}`
-    const data = {
-        from: 'admin@learndatabases.dev',
-        to: receiver,
-        subject: 'Congratulations!',
-        text: 'Welcome to C0D3',
-        html: `
+mgModule.sendPasswordResetEmail = (receiver, token) => {
```

Comment:
Since this name has changed, one in the test should be changed too.

Reply:
good catch

Reply:
@greenc123 https://github.com/garageScript/databases/blob/master/services/mailer.test.js#L29